handbook/engineering/index.md

```diff
@@ -70,6 +70,18 @@ Our engineering organization is divided into mission based teams that contain th
 
 Start a discussion with your manager if you are interested to switch teams. Your manager will chat with you to understand what you want and then propose next steps. Team transfers need to be approved by the [VP of Engineering](roles.md#vp-engineering).
 
+### Communication
+
+#### Slack channel convention
+
+Engineering teams follow the following convention when creating Slack channels for both internal (own team) and external (all teammates) communication.
+
+1. ${TEAMNAME}-team for internal team stuff. E.g. `web-team`
```

Comment:
Could we define "internal team stuff" and "external team communication" more clearly? I am worried that this leaves a lot of room to interpretation, and communication will drift to be mostly in the `-team` channels because they are argued to all be "internal team stuff". E.g. is a discussion around a decision towards the team's goals or an implementation discussion "internal team stuff", or "external team communication"? They could be argued to be internal, but I think as a company we profit from keeping those discussions in the external channels where people can subscribe if they are interested and chime in if they have something valuable to contribute. If we don't call this out here explicitly (in accordance with our openness value) I'm worried new teammates (coming from other companies with different cultures) will assume that the external channel is only for  _announcements_ to the wider company and everything else becomes effectively _internal by default_, when it should be the other way around (external/open by default).

The sentence below about "all work related conversation happens in the open in their `-chat` channel" does say this, but it could be missed easily and the two bullet points here don't intuitively align with that statement.

Reply:
@felixfbecker I updated the copy slightly to convey this message. Let me know if you think it still needs further tweaking.